apps/wing/src/wingc.ts

```diff
@@ -172,14 +173,13 @@ export async function load(options: WingCompilerLoadOptions) {
     preopens,
   });
 
-  const importObject = {
+  const importObject = merge({
```

Comment:
not sure it's worth introducing a package for this. Arguable the API should just be simpler here (we should accept an explicit function to set for new_diagnostic, not the entire import object).

Reply:
Originally I thought I might have more than one import function in the env. So I needed merge to avoid some ugliness here. But since I got rid of `send_notification` I think I can revert this.